deployment/united-manufacturing-hub/templates/softing/deployment.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: apps/v1
```

Comment:
do not put this into the helm chart. the goal is to have a tutorial containing a kubernetes deployment file like this and to add it then after UMH installation of the stack.

For our customer, I can do this in the setup scripts